### DISCUSSION TITLE
Adding additional light source

### DISCUSSION THREAD

**Discussion by snapo** [google/model-viewer#3808] [Q&A] [answered]
Hi,
Is it somehow possible to add an additional lightsource?
My Model does not contain any light source, and therefore is very dark and unreadable...
![image](https://user-images.githubusercontent.com/6347922/190522422-1db6221a-dbb3-4ee7-a6dc-2b36b531cd17.png)

**Comment by elalish** [comment, marked as answer]
The simplest way is to increase [`exposure`](https://modelviewer.dev/docs/#entrydocs-lightingandenv-attributes-exposure). You can also set your own HDR environment lighting: https://modelviewer.dev/examples/lightingandenv/#environmentLighting
